runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -933,6 +939,7 @@ impl<T: Config> Module<T> {
             ProposalDetails::UpdateChannelPayouts(..) => {
                 T::UpdateChannelPayoutsProposalParameters::get()
             }
+            ProposalDetails::FreezePallet(..) => T::SetInvitationCountProposalParameters::get(),
```

Comment:
```suggestion
            ProposalDetails::FreezePallet(..) => T::FreezePalletProposalParameters::get(),
```

Reply:
Also please "export" the proposal parameters as a runtime pallet constant in `decl_module! { ... }` like other proposal, as it is needed by pioneer to display and in the integration [tests](https://github.com/freakstatic/joystream/pull/1#discussion_r1379691315).